### ISSUE TITLE
Use shapley to ensure there are no miss-labeled  datas in the test/validation set

### ISSUE THREAD

**Issue by Thomas-Galtier** [SubstraFoundation/distributed-learning-contributivity#50]
Usually the test set and the validation set are selected at random among the available data, but what if this method selects miss-labeled or tainted data in this test/validation set? I think we could use the Shapley value to detected miss-labeled datas in the test/validation set.

We call the training set Tr1 and the test/validation set V1.
The idea would be to use a different test/validation dataset  V2 and to do a training with a  dataset Tr2 such that V1 is in Tr2. This way we can compute the  contributivity score of the data of V1 and use it to identify miss-labelled data.
